test-cases/tests/alerts/c18-validate-api-usage-alerts.md

```diff
@@ -38,146 +38,230 @@ tags:
 
 ## Steps
 
-1. Validate that rate-limiting alerts ConfigMap has been created
+1.  Validate that rate-limiting alerts ConfigMap has been created
 
-   ```sh
-   oc get cm rate-limit-alerts -n redhat-rhoam-operator
-   ```
+    ```shell script
+    oc get cm rate-limit-alerts -n redhat-rhoam-operator
+    ```
 
-2. Verify that level1, level2 and level3 rate-limiting alerts are present
+2.  Verify that level1, level2 and level3 rate-limiting alerts are present
 
-   ```sh
-   oc get prometheusrules -n redhat-rhoam-marin3r
-   ```
+    ```shell script
+    oc get prometheusrules -n redhat-rhoam-marin3r
+    ```
 
-3. Modify the `sku-limits-managed-api-service` configmap to set the rate limit to 100 requests per minute.
-   Run
+3.  Modify the `sku-limits-managed-api-service` configmap to set the rate limit to 100 requests per minute.
+    Run
 
-   ```sh
-   oc patch configmap sku-limits-managed-api-service -n redhat-rhoam-operator -p '"data": {        "rate_limit": "{\n  \"RHOAM SERVICE SKU\": {\n    \"unit\": \"minute\",\n    \"requests_per_unit\": 100\n  }\n}"    }'
-   ```
+    ```shell script
+    oc patch configmap sku-limits-managed-api-service -n redhat-rhoam-operator -p '"data": {        "rate_limit": "{\n  \"RHOAM SERVICE SKU\": {\n    \"unit\": \"minute\",\n    \"requests_per_unit\": 100,\n   \"soft_daily_limits\": [\n      5000000,\n      10000000,\n      15000000\n    ]\n }\n}"    }'
+    ```
 
-4. Modify the `rate-limit-alerts` to allow alerts to fire on a per minute basis:
+4.  Modify the `rate-limit-alerts` to allow alerts to fire on a per minute basis:
 
-   ```sh
-   oc patch configmap rate-limit-alerts -n redhat-rhoam-operator -p '"data": {
-       "alerts": "{\n  \"api-usage-alert-level1\": {\n    \"ruleName\": \"RHOAMApiUsageLevel1ThresholdExceeded\",\n    \"level\": \"warning\",\n    \"minRate\": \"80%\",\n    \"maxRate\": \"90%\",\n    \"period\": \"1m\"\n  },\n  \"api-usage-alert-level2\": {\n    \"ruleName\": \"RHOAMApiUsageLevel2ThresholdExceeded\",\n    \"level\": \"warning\",\n    \"minRate\": \"90%\",\n    \"maxRate\": \"95%\",\n    \"period\": \"1m\"\n  },\n  \"api-usage-alert-level3\": {\n    \"ruleName\": \"RHOAMApiUsageLevel3ThresholdExceeded\",\n    \"level\": \"warning\",\n    \"minRate\": \"95%\",\n    \"period\": \"1m\"\n  }\n}"
-   }'
-   ```
+    ```shell script
+    oc patch configmap rate-limit-alerts -n redhat-rhoam-operator -p '"data": {
```

Comment:
This `oc patch` command needs updating to reflect the latest configmap which includes the `rate-limit-spike` block. For example:

```
data:
  alerts: |-
    {
      "api-usage-alert-level1": {
        "type": "Threshold",
        "level": "info",
        "ruleName": "RHOAMApiUsageLevel1ThresholdExceeded",
        "period": "4h",
        "threshold": {
          "minRate": "80%",
          "maxRate": "90%"
        }
      },
      "api-usage-alert-level2": {
        "type": "Threshold",
        "level": "info",
        "ruleName": "RHOAMApiUsageLevel2ThresholdExceeded",
        "period": "2h",
        "threshold": {
          "minRate": "90%",
          "maxRate": "95%"
        }
      },
      "api-usage-alert-level3": {
        "type": "Threshold",
        "level": "info",
        "ruleName": "RHOAMApiUsageLevel3ThresholdExceeded",
        "period": "30m",
        "threshold": {
          "minRate": "95%"
        }
      },
      "rate-limit-spike": {
        "type": "Spike",
        "level": "warning",
        "ruleName": "RHOAMApiUsageOverLimit",
        "period": "30m"
      }
    }
```